src/conf/data/audio_MNIST.yaml

```diff
@@ -4,6 +4,9 @@ data:
   num_workers: 10
   sample_rate: 8000
   data_sample_rate: 8000
+  max_freq: 4000
+  n_fft: 256
+  hop_length: 64
   train_dataset:
     _target_: data.ClassifierDataset
     data_path: 'PATH_TO_AUDIO_MNIST_DATA'
```

Comment:
I know it's not part of the pr, but what?

Reply:
I changed the yaml structure such that these parameters are in data and not in preprocessing. Makes way more sense.

Reply:
I meant this:
`data_path: 'PATH_TO_AUDIO_MNIST_DATA'`
:)

Reply:
oh, on purpose I don't have paths to data, cause it can by anywhere on user machine